rskj-core/src/main/java/co/rsk/net/sync/SyncConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package co.rsk.net.sync;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.annotation.concurrent.Immutable;
+import java.time.Duration;
+
+@Immutable
+public final class SyncConfiguration {
+    public static final SyncConfiguration DEFAULT = new SyncConfiguration(5, 60, 10, 10, 20, 192);
+
+    @VisibleForTesting
+    public static final SyncConfiguration IMMEDIATE_FOR_TESTING = new SyncConfiguration(1, 1, 3, 1, 5, 192);
```

Comment:
/?/?

Reply:
It's a fixed conf to make some tests easier to build